etc/kayobe/kolla.yml

```diff
@@ -205,6 +205,64 @@ stackhpc_third_party_repos:
   - url: "{{ stackhpc_repo_treasuredata_4_url }}"
     file: "td.repo"
 
+# This section still seems to have issues
+
+stackhpc_rocky_9_repos:
+  - url: "{{ stackhpc_repo_rocky_9_baseos_url }}"
+    file: "rocky.repo"
+  - url: "{{ stackhpc_repo_rocky_9_appstream_url }}"
+    file: "rocky-appstream.repo"
+  - url: "{{ stackhpc_repo_rocky_9_extras_url }}"
+    file: "rocky-extras.repo"
+
+stackhpc_rocky_9_additional_repos:
+  - url: "{{ stackhpc_repo_centos_stream_9_nfv_openvswitch_url }}"
+    file: "CentOS-NFV-OpenvSwitch.repo"
+  - url: "{{ stackhpc_repo_centos_stream_9_opstools_url }}"
+    file: "CentOS-NFV-opstools.repo"
+  - url: "{{ stackhpc_repo_rocky_9_crb_url }}"
+    file: "rocky-crb.repo"
+  - url: "{{ stackhpc_repo_rocky_9_highavailability_url }}"
+    file: "rocky-highavailability.repo"
+
+stackhpc_rocky_9_third_party_repos:
+  - url: "{{ stackhpc_repo_grafana_url }}"
+    file: "grafana.repo"
+  - url: "{{ stackhpc_repo_rhel_9_treasuredata_4_url }}"
+    file: "td.repo"
+  - url: "{{ stackhpc_repo_rhel_9_mariadb_10_6_url }}"
+    file: "mariadb.repo"
+  - url: "{{ stackhpc_repo_rhel_9_influxdb_url }}"
+    file: "influxdb.repo"
+  - url: "{{ stackhpc_repo_centos_stream_9_storage_ceph_quincy_url }}"
+    file: "ceph.repo"
+  - url: "{{ stackhpc_repo_rabbitmq_erlang_url }}"
+    file: "erlang.repo"
+  - url: "{{ stackhpc_repo_rabbitmq_server_url }}"
+    file: "rabbitmq.repo"
+
+stackhpc_RHEL_9_repos:
+  - url: "{{ stackhpc_repo_epel_9_url }}"
+    file: "epel.repo"
+
+base_centos_repo_overrides_post_yum_8: |
+  {# fixme #}
+      && \
+  {% for repo in stackhpc_centos_stream_repos + stackhpc_epel_repos + stackhpc_centos_additional_repos + stackhpc_third_party_repos %}
```

Comment:
nit: it would be more DRY to make a variable for the repo list

Reply:
Done